data/lammps_input_example.lammps

```diff
@@ -0,0 +1,31 @@
+log log.si-${T}-${S}.lammps
+
+units           metal
+atom_style      atomic
+atom_modify     map array
+
+lattice         diamond 5.43
+region          simbox block    0 ${S} 0 ${S} 0 ${S}
+create_box      1 simbox
+create_atoms    1 region simbox
+
+#read_dump ${DUMP} ${STEP} x y z vx vy vz fx fy fz box yes replace no purge yes add yes
+
+mass 1 28.0855
+
+group Si type 1
+
+pair_style sw
+pair_coeff * * ../Si.sw Si
```

Comment:
This example won't run without the input file `Si.sw`. I think we should add that to the repo as well, given that it's a tiny text file.